src/styles/blocks/page.scss

```diff
@@ -0,0 +1,51 @@
+.page {
+  font-family: "Poppins", "Inter", sans-serif;
+  scroll-behavior: smooth;
+
+  &__menu {
+    position: fixed;
+    top: 0;
+    left: 0;
+    right: 0;
+
+    transform: translateX(-100%);
+    transition: transform 0.3s;
+
+    &:target {
+      transform: translateX(0);
+    }
+  }
+
+  &__body--with-menu {
+    overflow: hidden;
+  }
+
+  &__main {
+    background: $second-text-colour;
+    padding-bottom: 120px;
+
+    @include tablet {
+      padding-bottom: 134px;
+    }
+
+    @include desktop {
+      padding-bottom: 180px;
+    }
+  }
+
+  &__section-title {
+    font-family: "Poppins", sans-serif;
```

Comment:
don't add `font-family` property in every css rule. You have already included this property to entire page in `.page class`